modules/ossm-add-project-using-label-selectors-cli.adoc

```diff
@@ -4,14 +4,14 @@
 
 :_content-type: PROCEDURE
 [id="ossm-adding-project-using-label-selectors-cli_{context}"]
-= Adding a project to the {SMProductShortName} using label selectors with the CLI 
+= Adding a project to the mesh using label selectors with the CLI 
 
 You can use label selectors to add a project to the {SMProductShortName} with the CLI.
 
 .Prerequisites
-* The deployment has an installed, verified {SMProductName} Operator.
+* You have installed the {SMProductName} Operator.
 * The deployment has an existing `ServiceMeshMemberRoll` resource.
-* You are logged in as a user with mesh admin privileges.
+* You are logged in as a user with the `cluster-admin` role. If you use Red Hat OpenShift Dedicated, you are logged in as a user with the `dedicated-admin` role.
```

Comment:
```suggestion
* You are logged in as a user with the `cluster-admin` role. If you use Red Hat OpenShift Dedicated, you are logged in as a user with the `dedicated-admin` role.
```
- Should `OpenShift Dedicated` be an attribute?

Reply:
done